composer.json

```diff
@@ -18,19 +18,22 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
-        "symfony/framework-bundle": "^5.2 || ^6.0"
+        "php": ">7.4",
```

Comment:
Is the PHP 7.4 support dropping intentional?

Reply:
Sry typo, i will fix it before merging